parlai/core/torch_generator_agent.py

```diff
@@ -1114,15 +1117,21 @@ def _generate(
         encoder_states = model.reorder_encoder_states(encoder_states, inds)
         incr_state = None
 
+        total_decoder_elapsed_time = 0
+        num_decoder_forwards = 0
         for _ts in range(max_ts):
             if all((b.is_done() for b in beams)):
                 # exit early if possible
                 break
 
+            decoder_start_time = time.time()
```

Comment:
https://stackoverflow.com/questions/17579357/time-time-vs-timeit-timeit

fyi

Reply:
interesting to know!!!!

Reply:
good point, changing